cmd/ack-generate/command/common.go

```diff
@@ -0,0 +1,53 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package command
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// ensureDir makes sure that a supplied directory exists and
+// returns whether the directory already existed.
+func ensureDir(fp string) (bool, error) {
```

Comment:
Could this function's name be parallel to isDirWriteable (ex: isDirValid) or is it not because it returns both a bool and error?